tests/protocol/mockwiredata.py

```diff
@@ -377,17 +379,39 @@ def replace_xml_attribute_value(xml_document, element_name, attribute_name, attr
             raise Exception("Could not match attribute '{0}' of element '{1}'".format(attribute_name, element_name))
         return new_xml_document
 
-    def set_etag(self, etag):
-        '''
-        Sets the ETag for the mock response
-        '''
+    def set_etag(self, etag, timestamp=None):
```

Comment:
I need new test helpers to change the source (fast track vs fabric) and timestamp of the goal states in the mock data